src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,20 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment, key }) => (
+  <div className="CommentInfo" key={comment.id}>
```

Comment:
```suggestion
  <div className="CommentInfo"}>
```

there is no need to set `key` you already set in `CommentList`